tests/functional/utils.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from inspire_relations import command_producers
+
+from uuid import UUID
+
+from neo4j.v1.session import Session
+
+
+def create_record_stub(json, uuid):
+    """
+    Creates invenio-records Record's stub with RecordMetadata stub inside.
+    """
+    class RecordMetadataStub:
+        def __init__(self, uuid):
+            self.id = UUID(uuid)
+
+    class RecordStub(dict):
+        def __init__(self, json, uuid):
+            self.model = RecordMetadataStub(uuid)
+            dict.__init__(json)
+
+    return RecordStub(json, uuid)
+
+
+def are_dicts_the_same(first_dict, second_dict):
+    return set(first_dict.items()) == set(second_dict.items())
```

Comment:
I thought `==` for the `dicts` has different behaviour. Anyway, thanks for pointing it out!